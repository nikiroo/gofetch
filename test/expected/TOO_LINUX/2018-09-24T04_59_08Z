         CLIP OS, LE SYSTÈME D’EXPLOITATION "MADE IN FRANCE"       

  2018/09/24

  o News link: https://www.toolinux.com/?CLIP-OS-le-systeme-d-exploitation-made-in-France
  o Source link: 


    CLIP OS, le système d’exploitation "made in France"
    
    lundi 24 septembre 2018
    
    Maintenu  par  l’ANSSI,  CLIP  OS  est  un  projet  de  système
    d’exploitation  open  source  basé  sur  un  noyau Linux et des
    logiciels  libres.  C’était  une  idée  depuis  dix  ans, c’est
    désormais un projet bien réel.
    
    Initialement   développé   pour   répondre   aux   besoins  des
    administrations,  les précédentes versions de CLIP OS n’étaient
    pas  disponibles  publiquement.  Elles le sont depuis septembre
    2018   :   «  Le  projet  se  base  sur  plus  de  dix  ans  de
    [1]développement  interne  à  l’ANSSI  pour  créer  un  système
    d’exploitation durci. »
    
    Attention,   CLIP   OS   n’est   pas  "souverain"  :  «  Si  le
    développement  du  projet  est initié et réalisé par des agents
    de  l’ANSSI,  la  majeure  partie  du  code  source est issu de
    projets  open  source  (noyau  Linux, suite de compilation GCC,
    etc.).  Le  code  source  de  CLIP  OS est aujourd’hui ouvert à
    tous.  Le  projet  est  basé  sur  Gentoo  Hardened et suit des
    principes similaires à Chromium OS ou au projet Yocto. »
    
    Deux  versions  du  projet  sont  publiées le code source et la
    documentation  (en français) de CLIP OS version 4, puis le code
    source  et  la documentation (en anglais) de CLIP OS version 5.
    Cette   version   est  en  cours  de  développement.  Elle  est
    considérée comme étant au statut d’alpha.
    
    Les  sources  du  projet CLIP OS [2]sont hébergées sur GitHub ,
    dans  l’organisation  CLIP OS. Les portions de code spécifiques
    au  projet  CLIP  OS  sont  mises  à disposition en open source
    (principalement sous licence LGPL 2.1+).
    
    Liens
    
    [3]Site web officiel
    
    [4]Présentation du projet CLIP OS
    
    [5]CLIP OS version 4
    
    [6]CLIP OS version 5
    
    [7]
    
    
    
    [1] https://www.ssi.gouv.fr/
    
    [2] https://github.com/CLIPOS/bugs
    
    [3] https://clip-os.org/fr/
    
    [4] https://www.ssi.gouv.fr/administration/services-securises/-
    clip/
    
    [5] https://clip-os.org/fr/clipos4
    
    [6] https://clip-os.org/fr/clipos5/
    
    [7] https://www.toolinux.com/?CLIP-OS-le-systeme-d-exploitatio-
    n-made-in-France#forum



