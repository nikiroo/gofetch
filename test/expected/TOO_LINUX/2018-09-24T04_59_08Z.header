0CLIP OS, le système d’exploitation "made in France"	null/TOO_LINUX/2018-09-24T04_59_08Z		70
i  2018/09/24
i
i    Maintenu  par  l’ANSSI,  CLIP  OS  est  un  projet  de  système
i    d’exploitation  open  source  basé  sur  un  noyau Linux et des
i    logiciels  libres.  C’était  une  idée  depuis  dix  ans, c’est
i    désormais  un  projet  bien  réel.  Initialement développé pour
i    répondre  aux  besoins  des  administrations,  les  précédentes
i    versions  de  CLIP  OS  n’étaient pas disponibles publiquement.
i    Elles le (...)
i
