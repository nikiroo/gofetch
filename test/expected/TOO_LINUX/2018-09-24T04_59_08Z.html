<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story'>
	<h1>CLIP OS, le système d’exploitation "made in France"</h1>
	<div class='details'>(2018/09/24)</div>
	<br/>
    <ul>
        <li>News link: <a href='https://www.toolinux.com/?CLIP-OS-le-systeme-d-exploitation-made-in-France'>https://www.toolinux.com/?CLIP-OS-le-systeme-d-exploitation-made-in-France</a></li>
        <li>Source link: <a href=''></a></li>
    </ul>
	<br/>
	<div class='content' style='text-align: justify'>
		CLIP OS, le système d’exploitation &quot;made in France&quot;<br/><br/>lundi 24 septembre 2018<br/><br/>Maintenu par l’ANSSI, CLIP OS est un projet de système d’exploitation open source basé sur un noyau Linux et des logiciels libres. C’était une idée depuis dix ans, c’est désormais un projet bien réel.<br/><br/>Initialement développé pour répondre aux besoins des administrations, les précédentes versions de CLIP OS n’étaient pas disponibles publiquement. Elles le sont depuis septembre 2018 : « Le projet se base sur plus de dix ans de [1]développement interne à l’ANSSI pour créer un système d’exploitation durci. »<br/><br/>Attention, CLIP OS n’est pas &quot;souverain&quot; : « Si le développement du projet est initié et réalisé par des agents de l’ANSSI, la majeure partie du code source est issu de projets open source (noyau Linux, suite de compilation GCC, etc.). Le code source de CLIP OS est aujourd’hui ouvert à tous. Le projet est basé sur Gentoo Hardened et suit des principes similaires à Chromium OS ou au projet Yocto. »<br/><br/>Deux versions du projet sont publiées le code source et la documentation (en français) de CLIP OS version 4, puis le code source et la documentation (en anglais) de CLIP OS version 5. Cette version est en cours de développement. Elle est considérée comme étant au statut d’alpha.<br/><br/>Les sources du projet CLIP OS [2]sont hébergées sur GitHub , dans l’organisation CLIP OS. Les portions de code spécifiques au projet CLIP OS sont mises à disposition en open source (principalement sous licence LGPL 2.1+).<br/><br/>Liens<br/><br/>[3]Site web officiel<br/><br/>[4]Présentation du projet CLIP OS<br/><br/>[5]CLIP OS version 4<br/><br/>[6]CLIP OS version 5<br/><br/>[7]<br/><br/><br/><br/>[1] https://www.ssi.gouv.fr/<br/><br/>[2] https://github.com/CLIPOS/bugs<br/><br/>[3] https://clip-os.org/fr/<br/><br/>[4] https://www.ssi.gouv.fr/administration/services-securises/clip/<br/><br/>[5] https://clip-os.org/fr/clipos4<br/><br/>[6] https://clip-os.org/fr/clipos5/<br/><br/>[7] https://www.toolinux.com/?CLIP-OS-le-systeme-d-exploitation-made-in-France#forum
	</div>
<hr/>
</div>
</body>
